quickstart/101-attestation-provider/providers.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+
+  required_version = ">=0.12"
+  
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "~>2.0"
+    }
```

Comment:
Can we add `random` provider constraint here please? It might look verbose for this built-in provider but it's a best practice to declare all providers' version constraints explicitly to avoid potential inconsistency introduced by different provider's versions.